src/components/UI/ResourcesCard/ResourcesCard.tsx

```diff
@@ -86,6 +99,25 @@ export const ResourcesCard: FunctionComponent<ResourcesCardProps> = ({ details }
         )}
         {dateTime && <div className="text-grey text-base font-medium pb-3">{dateTime}</div>}
         <p className="mb-4">{description}</p>
+        {youtubeEmbedCode && videoTimeStamps && (
+          <Dropdown
+            data-testid="quickVideoLinksDropdown"
+            dropdownButtonText="Quick Video Links"
+            className="rounded border border-grey-300 text-grey-700 p-2 mb-2"
+          >
+            {videoTimeStamps.map((eachTimeStamp, i) => {
+              return (
+                <DropdownItem
+                  key={i}
+                  data-testid="videoTimeStampsDropdown"
+                  onClick={() => setCurrentTimeStamp(eachTimeStamp.timeStamp)}
+                >
+                  {eachTimeStamp.title} [{convertSecondsToMinAndSec(eachTimeStamp.timeStamp)}]
```

Comment:
same, so it will be `videoChapter.timeStamp`

Reply:
ok